litmus-portal/backend/auth/router/router.go

```diff
@@ -5,16 +5,20 @@ import (
 	"github.com/gin-gonic/gin"
 
 	"github.com/litmuschaos/litmus/litmus-portal/backend/auth/controller"
+	github "github.com/litmuschaos/litmus/litmus-portal/backend/auth/controller/GithubUser"
 	"github.com/litmuschaos/litmus/litmus-portal/backend/auth/controller/user"
 )
 
 const (
-	loginRoute  = "/login"
-	updateRoute = "/update"
+	loginRoute     = "/login"
+	updateRoute    = "/update"
+	githubMidRoute = "/login/github"
+	oauthRoute     = "/oauth/github"
```

Comment:
why two routes for one login? can't we have a single route for like `/oauth/github` where client(UI) can do POST request with `code`?
just wanted to know.